specification/iothub/resource-manager/Microsoft.Devices/preview/2018-12-01-preview/iothub.json

```diff
@@ -2386,7 +2386,7 @@
           "minimum": 10485760
         },
         "encoding": {
-          "description": "Encoding that is used to serialize messages to blobs. Supported values are 'avro' and 'avrodeflate'. Default value is 'avro'.",
+          "description": "Encoding that is used to serialize messages to blobs. Supported values are 'avro' and 'avroDeflate'. Default value is 'avro'.",
```

Comment:
I am not sure if the service supports avro**D**eflate. Please revert this change.